src/cgroups.rs

```diff
@@ -55,7 +54,7 @@ impl Cgroups {
 }
 
 fn get_cgroups() -> Result<Vec<String>, std::io::Error> {
-    Ok(fs::read_dir("/sys/fs/cgroup")?
+    Ok(fs::read_dir("/sys/fs/cgroup/memory")?
```

Comment:
can you double-check on this? cgroup_v2 doesn't have `memory` dir on my system.

Reply:
it's for cgroup v1 since the enclave kernel has v1, not v2